pennylane/ops/op_math/linear_combination.py

```diff
@@ -14,7 +14,7 @@
 """
 LinearCombination class
 """
-# pylint: disable=too-many-arguments, protected-access
+# pylint: disable=too-many-arguments, protected-access, super-init-not-called, too-many-instance-attributes
```

Comment:
Why did the `super-init-not-called` warning have to be disabled? `super().__init__` is being called on line 149

Reply:
Remnant from a previous attempt, good catch!